homeassistant/components/esphome/strings.json

```diff
@@ -17,7 +17,7 @@
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
         },
-        "description": "Please enter connection settings of your [ESPHome](https://esphomelib.com/) node."
+        "description": "Please enter connection settings of your [ESPHome](https://esphome.io/) node."
```

Comment:
Usually we use placeholders for urls

Reply:
Yeah agreed, this need to be adjusted so we don't end up needing to update the translations in these cases

Reply:
I have no idea how that is supposed to be done. I see more URLs in other string.json files. Can you give a little hint?

Reply:
This gives an indication in Renault config flow: https://github.com/home-assistant/core/pull/55547/files

Reply:
The URL in the "description" is now using a placeholder. 
The URL in the "resolve_error" is not using "description_placeholders" so it cann't be a placeholder.

Reply:
I'm confused by this. Are you saying that `desciption_placeholders` do not work with errors?
I am not familiar with the frontend, but if that's the case then I would think it's a hole that needs pluging.

Reply:
Now, it's more that on this line the error is just a string,
https://github.com/home-assistant/core/blob/a53aaf696c04ac1fc03f7e77bce40e62d82acbfe/homeassistant/components/esphome/config_flow.py#L278-L279
And somehow that is translated to the string in the string.json
https://github.com/home-assistant/core/blob/a53aaf696c04ac1fc03f7e77bce40e62d82acbfe/homeassistant/components/esphome/strings.json#L9
But how to use `desciption_placeholders `there is a question to me.

Reply:
I guess there's three options:
- keep the URL inside the `strings.json`
- always add the url to `description_placeholders`, regardless of whether an error was present
- check if the `resolve_error` is present in `errors` and adjust the `desciption_placeholders` if needed

I am not sure which is the recommended way.